client/components/Price/Price.js

```diff
@@ -0,0 +1,38 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import styles from './Price.css';
+
+export default class Price extends PureComponent {
+  static propTypes = {
+    price: PropTypes.string,
+    onChange: PropTypes.func,
+  };
+
+  handleOnChangeAction = (e) => {
+    const { price } = this.props;
+    const value = price.length ? price.split(',') : [];
```

Comment:
when `price` is not passed over to this component since it is not required, this will throw an error

Reply:
did some changes here. please check thanks